### PR TITLE
Add model resources for ViT

### PR DESCRIPTION
**Pull request by stanleycai95**
# What does this PR do?

Adds resources on ViT according to https://github.com/huggingface/transformers/issues/20055

Fixes https://github.com/huggingface/transformers/issues/20055 (partially)

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@stevhliu

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks for adding those!

**Review by stevhliu — COMMENTED** [review]
Thanks for your contribution!

File: docs/source/en/model_doc/vit.mdx [hunk 1]
```diff
@@ -87,6 +87,33 @@ Note that we converted the weights from Ross Wightman's [timm library](https://g
 go to him!
 
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with ViT. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="image-classification"/>
```
[the remark this answers is not in the document]
Comment (stanleycai95) [reply docs/source/en/model_doc/vit.mdx:95]: Makes sense - there are quite a few image classification resources for ViT, I updated the format. Please definitely let me know if I need to make further changes.

**Review by stevhliu — APPROVED** [review]
Looks great, thanks for your contribution @stanleycai95! 🤗

**Pull request merged**